src/api/instance-methods.md

```diff
@@ -121,7 +121,9 @@
 
 - **Option: deep**
 
-  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. Note that you don't need to do so to listen for array mutations.
+  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. Note that in Vue3 you **also** need to do so to listen for array mutations.
```

Comment:
**suggestion**: this is a documentation for v3, I don't think we need an additional reminder about the current version here. WDYT about rephrasing:

```suggestion
  To also detect nested value changes inside Objects, you need to pass in `deep: true` in the options argument. This option also can be used to watch array mutations.
```

Reply:
agree with you

Reply:
@likev If you're happy with the change could you please merge it into your branch? You should see a button beneath the suggested change that will commit it for you. Thanks.

Reply:
There are a lot I need to learn about github, thanks!